pkg/log/log.go

```diff
@@ -87,15 +87,21 @@ func MakeLogger(logger log.Logger) *FilteredLogger {
 	if verbosityFlag := flag.Lookup("v"); verbosityFlag != nil {
 		defaultVerbosity, _ = strconv.Atoi(verbosityFlag.Value.String())
 	} else {
-		defaultVerbosity = 0
+		// "the practical default level is V(2)"
+		// see https://github.com/kubernetes/community/blob/master/contributors/devel/logging.md
```

Comment:
Ha, great that you found that. I was looking for something like that several times!